tests/BenchmarkDotNet.Tests/Portability/Cpu/WmicCpuInfoParserTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using BenchmarkDotNet.Portability.Cpu;
+using Xunit;
+
+namespace BenchmarkDotNet.Tests.Portability.Cpu
+{
+    public class WmicCpuInfoParserTests
+    {
+        [Fact]
+        public void EmptyTest()
+        {
+            var parser = new WmicCpuInfoParser(string.Empty);
+            Assert.Equal(null, parser.ProcessorName);
+            Assert.Equal(null, parser.PhysicalProcessorCount);
+            Assert.Equal(null, parser.PhysicalCoreCount);
+            Assert.Equal(null, parser.LogicalCoreCount);
+        }
+
+        [Fact]
+        public void MalformedTest()
+        {
+            var parser = new WmicCpuInfoParser("malformedkey=malformedvalue\n\nmalformedkey2=malformedvalue2");
+            Assert.Equal(null, parser.ProcessorName);
+            Assert.Equal(null, parser.PhysicalProcessorCount);
+            Assert.Equal(null, parser.PhysicalCoreCount);
+            Assert.Equal(null, parser.LogicalCoreCount);
+        }
+
+        [Fact]
+        public void RealTwoProcessorEightCoresTest()
+        {
+            #region cpuinfo
+
+            var cpuInfo = @"
+
+Name=Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz
+NumberOfCores=8
+NumberOfLogicalProcessors=16
+
+
+Name=Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz
+NumberOfCores=8
+NumberOfLogicalProcessors=16
+
+";
+
+            #endregion
+
+            var parser = new WmicCpuInfoParser(cpuInfo);
+            Assert.Equal("Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz", parser.ProcessorName);
+            Assert.Equal(2, parser.PhysicalProcessorCount);
+            Assert.Equal(16, parser.PhysicalCoreCount);
+            Assert.Equal(32, parser.LogicalCoreCount);
+        }
+
+        [Fact]
+        public void RealOneProcessorFourCoresTest()
+        {
+            #region cpuinfo
```

Comment:
Why do we need `#region` here? Could we remove it?

Reply:
Fixed